website/public/locales/en/message.json

```diff
@@ -23,5 +23,7 @@
   "submit_labels": "Submit",
   "tree_stopped": "Tree stopped {{id}}",
   "view_user": "View user",
-  "your_recent_messages": "Your Recent Messages"
+  "your_recent_messages": "Your Recent Messages",
+  "synthetic": "Synthetic",
+  "synthetic_explain": "This message is generated by our AI"
```

Comment:
I think when people read "our" they will start to think  and ask if the models are ready for public use, which is not the case.

maybe we can remove "our"? leave it vague? This message is AI generated.

Reply:
done.